index.html

```diff
@@ -53,6 +53,312 @@ <h1 class="heading">Hey there. I’m Sean John.<br /> <span id="green-text">I’
+                                <path d="M46.6256 16.6706L43.4178 14.8185C42.9901 14.5716 42.4546 14.8406 42.2756 15.3153L40.8398 21.4238L39.2003 20.4772C38.7013 20.1891 38.0528 20.4892 37.9746 21.1185L37.344 29.6189C37.3614 29.9181 37.7595 30.0516 37.8552 29.7214L39.9916 24.2101L42.4866 25.6506C42.9143 25.8975 43.408 25.7008 43.5871 25.2261L46.8917 17.6914C47.1003 17.33 46.9821 16.8764 46.6256 16.6706Z" fill="#091E42"/>
+                                </g>
+                                <defs>
+                                <clipPath id="clip0_34_2481">
+                                <rect width="48" height="48" fill="white"/>
+                                </clipPath>
+                                </defs>
+                                </svg>
+                                
+                        </div>
+                        <div class="quality-content">
+                            <div class="quality-name">Skills</div>
+                            <ul class="skill-tags">
+                                <li class="skill-tag">Codekit</li>
+                                <li class="skill-tag">GitHub</li>
+                                <li class="skill-tag">Codepen</li>
+                                <li class="skill-tag">Codepen</li>
+                                <li class="skill-tag">Codepen</li>
+                            </ul>
+                        </div>
+                        
+
+
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <div class="section contact-section">
+            <div class="container contact">
+                <div class="contact-info">
+                    I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                </div>
+
+                <div class="contact-form">
+                    <form class="form-block" action="submit">
+                        <div class="input-grid">
+                            <input placeholder="Name" type="text">
+                            <input placeholder="Last Name" type="text">
+                        </div>
+                        <input placeholder="Email Address" type="email" name="email" id="email">
+                        <textarea placeholder="Message" name="textarea" id="textarea" cols="30" rows="6"></textarea>
+                        <div class="button-group">
+                            <button class="see-project" type="submit">Get in touch</button>
+                        </div>
+                    </form>
+                </div>
+            </div>
+        </div>
```

Comment:
- [ ] Please ensure that you stick to the design guideline as much as possible
- [ ] The design only has one input field for `fullname`. Please only use one name input
- [ ] Please ensure that the heading text is aligned to the center like that of the original design
- [ ] Please kindly include the icons and email that is below the form in the original design

### Your Design
![image](https://user-images.githubusercontent.com/43995830/165323579-ae572a01-9d0c-4d12-8f3d-8a9074b07678.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/165323860-277569c6-9fde-4576-a8bc-d7629ddadf4d.png)